tutorials/codegen.ipynb

```diff
@@ -480,66 +480,66 @@
     "        self.frame = frame_codegen\n",
     "        # Can be used to dispatch other code generators for allocation/nodes\n",
     "        self.dispatcher = frame_codegen.dispatcher\n",
-    "        \n",
+    "\n",
     "        ################################################################\n",
-    "        # Register handlers/hooks through dispatcher: Can be used for \n",
+    "        # Register handlers/hooks through dispatcher: Can be used for\n",
     "        # nodes, memory copy/allocation, scopes, states, and more.\n",
-    "        \n",
+    "\n",
     "        # In this case, register scopes\n",
     "        self.dispatcher.register_map_dispatcher(dace.ScheduleType.LoopyLoop, self)\n",
-    "        \n",
+    "\n",
     "        # You can similarly use register_{array,copy,node,state}_dispatcher\n",
-    "        \n",
-    "    # A scope dispatcher will trigger a method called generate_scope whenever \n",
+    "\n",
+    "    # A scope dispatcher will trigger a method called generate_scope whenever\n",
     "    # an SDFG has a scope with that schedule\n",
-    "    def generate_scope(self, sdfg: dace.SDFG, scope: ScopeSubgraphView,\n",
+    "    def generate_scope(self, sdfg: dace.SDFG, dfg, scope: ScopeSubgraphView,\n",
```

Comment:
Add type hint

Reply:
Note that it should be `cfg`, not `dfg` anyway: https://github.com/spcl/dace/blob/main/dace/codegen/targets/target.py#L81